modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/BasicSubscription.java

```diff
@@ -77,8 +80,6 @@ public BasicSubscription(String subscriptionAsStr) {
                 this.subscriptionID = tokens[1];
```

Comment:
It would be more readable if the split string is put into variable before using as below (like String key = tokens[0]; )